client/src/modules/auth/graphql/mutations.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from '@apollo/client';
+
+export const deleteMe = gql`
+  mutation deleteMe($userId: Int!) {
+    deleteMe(id: $userId) {
```

Comment:
```suggestion
  mutation deleteMe {
    deleteMe {
```
By giving this an id parameter it implies that you can delete a user with that id.  That's not the case, so we should remove the parameter.